source/run/include/TG4RunConfiguration.h

```diff
@@ -109,18 +111,24 @@ class TG4RunConfiguration
   Bool_t IsSpecialControls() const;
   Bool_t IsSpecialCuts() const;
   Bool_t IsMTApplication() const;
+  Bool_t UseExternalGeometryConstruction() const;
+  Bool_t UseExternalParticleGeneration() const;
 
  protected:
   // data members
-  TString fUserGeometry;            ///< way of building geometry
-  TString fPhysicsListSelection;    ///< physics list selection
-  TString fSpecialProcessSelection; ///< special process selection
-  Bool_t fSpecialStacking;          ///< option for special stacking
-  Bool_t fMTApplication;            ///< option for MT mode if available
-  Bool_t fSpecialControls;          ///< option for special controls
-  Bool_t fSpecialCuts;              ///< option for special cuts
-  G4UImessenger* fAGDDMessenger;    //!< XML messenger
-  G4UImessenger* fGDMLMessenger;    //!< XML messenger
+  TString fUserGeometry; ///< way of building geometry
+  Bool_t
+    fUseExternalGeometryConstruction; ///< Rely on geometry is built externally
```

Comment:
These data/methods (*UseExternalGeometryConstruction*, *UseExternalParticleGeneration*) are already present in TVirtualMC; adding them to this class is redundant.
Could you look at this if they can be removed from here?